clients/banking/src/components/TransferWizardBeneficiary.tsx

```diff
@@ -33,45 +38,75 @@ const styles = StyleSheet.create({
 });
 
 type Props = {
+  accountCountry: AccountCountry;
+  accountId: string;
   initialBeneficiary?: Beneficiary;
   onSave: (beneficiary: Beneficiary) => void;
 };
 
-export const TransferWizardBeneficiary = ({ initialBeneficiary, onSave }: Props) => {
-  const [iban, setIban] = useState<string | undefined>(undefined);
-  const { data } = useUrqlQuery(
-    {
-      query: GetIbanValidationDocument,
-      pause: iban == undefined,
-      variables: {
-        // `pause` gives us the guarantee we get a valid iban
-        iban: iban as string,
-      },
-    },
-    [iban],
-  );
+export const TransferWizardBeneficiary = ({
+  accountCountry,
+  accountId,
+  initialBeneficiary,
+  onSave,
+}: Props) => {
+  const {
+    data: ibanVerification,
+    query: queryIbanVerification,
+    reset: resetIbanVerification,
+  } = useDeferredUrqlQuery(GetIbanValidationDocument);
+  const {
+    data: beneficiaryVerification,
+    query: queryBeneficiaryVerification,
+    reset: resetBeneficiaryVerification,
+  } = useDeferredUrqlQuery(GetBeneficiaryVerificationDocument);
```

Comment:
I think we could add a `useDebounce` over the query functions to avoid hitting the surepay API on every keystroke